docker/jdk8u/x86_64/ubuntu/Dockerfile

```diff
@@ -56,3 +56,4 @@ CMD ["images"]
 
 ARG OPENJDK_VERSION
 ENV OPENJDK_VERSION=$OPENJDK_VERSION
+ENV JDK_PATH=j2sdk-image
```

Comment:
Just a naming thing, this isn't really JDK_PATH but more the path to the JDK image created by the OpenJDK build process, so `JDK_IMAGE_PATH` or in those lines would be helpful, otherwise makes me think its like `JAVA_HOME` in a way.

Reply:
Yeah, JDK_PATH existed before this change, I just put it into the Dockerfile rather than passing it in from the scripts above (which have to set it differently depending on jdk8 or jdk9). I found it easier to just set it directly in the Dockerfile because it knows which jdk it's building.

Reply:
Okay no worries, one of these days I'll clean sweep such namings, as there are so many terms to remember in the OpenJDK world, easier to remember the better.